designer_v2/lib/features/forms/form_view_model.dart

```diff
@@ -195,7 +195,10 @@ abstract class FormViewModel<T> implements IFormGroupController {
     for (final entry in form.controls.entries) {
       final controlName = entry.key;
       final control = entry.value;
-      final isEnabledByDefault = _defaultControlStates[controlName] ?? true;
+      final isEnabledByDefault = _defaultControlStates[controlName];
+      if (isEnabledByDefault == null) {
+        return;
```

Comment:
This change leads to the following bug:

1. Create a new survey
2. Create a new yes/no question
3. Save question
4. Save survey

Now a dialog is shown that says that there are unsaved changes which should not be shown.

However, without this change the locking feature does not work properly.

Reply:
Can you find a workaround?

Reply:
Fixed in https://github.com/hpi-studyu/studyu/pull/558/commits/79062510633d5d196b42afdbb26d8cf0e91eddf2